content/blog/posts/o3de-ux-past-present-future.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Explore the Past, Present, & Future of O3DE's First-time User Experience"
+description: "Learn about how Amazon Web Services developed the first-time user experience for Open 3D Engine (O3DE) from Bhanuja Sanghavi, one of the UX designers contributing to O3DE."
+date: 2022-01-14
+slug: o3de-ftue-blog-post-1
+author: Bhanuja Sanghavi, Amazon Web Services
+blog_img: "/images/blog/announcement_thumbnail.jpg"
+full_img: "/images/blog/bjds-logo.jpg"
+---
+
+Learn about how Amazon Web Services developed the first-time user experience for Open 3D Engine (O3DE) from Bhanuja Sanghavi, one of the UX designers contributing to O3DE.
+
+## Explore the Past, Present, & Future of O3DE's First-time User Experience
+
+​
+Hello, I'm Bhanuja! Since September 2020, I have had the opportunity to work as a User Experience (UX) Design Intern for Amazon on its game engine Lumberyard, the predecessor to Open 3D Engine (O3DE). Over the past year, I have worked closely with the UX, Engineering, and Product teams to research the engine's First-time User Experience (FTUE), as well as the solutions to improve the ease and efficiency with which users set up, learn, and create with O3DE.
+​
+From the instant that game developers and content creators install the engine for the first time to the moment they begin to dive deep into O3DE's distinctive feature set, the FTUE is essential for shaping the user's path and, eventually, their success in building with O3DE. I'll explain what I mean.
+​
+
+### Why does First-time User Experience Matter?
+
+​
+As humans, we tend to make snap judgements; our brains have evolved to do so. Why? So we can piece together information into understandable groups, and to help us avoid making poor decisions.
+​
+What does this have to do with the first-time user experience? Simply put, users take less than a second to form their first opinion of a product, with only a few more seconds to reinforce this first impression. FTUE *is* the comprehensive set of thoughts, insights, and feelings that users experience through their first interaction with a product.
+​
+We interpret FTUE as beginning with the engine installation and project creation. An assessment of the engine's capability through its tools and workflows follows closely behind. Finally, the building of an initial game concept completes the FTUE. It is a make-or-break moment for new users, even more so than any other moment of the user's journey.
+​
+As the UX team, we are responsible for creating experiences that are intuitive, correspond with user requirements, and are representative of countless user workflows. We are constantly researching and advocating for the needs of our users, so they have a smooth and delightful first-time experience. This allows users to quickly see the value of O3DE, increase their productivity, and unleash their creativity. Ultimately, this boosts the adoption rate at which users select O3DE to build their first game or simulation project.
+​
+
```

Comment:
Can we add some additional line spaces here? The gap between the paragraph above to this heading is too cramped.